extension/package.json

```diff
@@ -3,17 +3,18 @@
   "version": "9.7.0",
   "description": "Integration between commercetools and Adyen payment service provider",
   "license": "MIT",
+  "type": "module",
   "scripts": {
     "check-coverage": "nyc check-coverage --lines 89",
     "test": "npm run check && npm run cover:unit && npm run cover:integration && npm run cover:e2e && npm run check-coverage",
     "cover:unit": "nyc --silent npm run unit",
     "cover:integration": "nyc --silent --no-clean npm run integration",
     "cover:e2e": "nyc --silent --no-clean npm run e2e",
-    "unit": "nyc mocha --file ./test/unit/mock-default-confg.js \"test/unit/**/*.spec.js\"",
-    "integration": "nyc mocha --file ./test/initial-integration-test-set-up.js \"test/integration/**/*.spec.js\"",
+    "unit": "nyc mocha --experimental-json-modules --file ./test/unit/mock-default-confg.js \"test/unit/**/*.spec.{cjs,js}\"",
```

Comment:
Where did you get this mocha flag for `--experimental-json-modules`? I could not find any docs for it in mocha.